specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/instancePoolUsages.json

```diff
@@ -0,0 +1,343 @@
+    "InstancePoolUsageListResult": {
+      "description": "A list of instance pool usages.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstancePoolUsage"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "InstancePoolUsageProperties": {
+      "description": "Properties of an instance pool usage.",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User-readable name of the metric.",
+          "type": "string",
+          "readOnly": true
+        },
+        "unit": {
+          "description": "Unit of the metric",
+          "enum": [
+            "VCores",
+            "Gigabytes"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "InstancePoolUsageUnit",
+            "modelAsString": true
+          }
+        },
+        "currentValue": {
+          "format": "int32",
+          "description": "Instance pool current value.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "currentLimit": {
+          "format": "int32",
+          "description": "Instance pool current limit.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "requestedLimit": {
+          "format": "int32",
+          "description": "Instance pool requested limit.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "managedInstanceUtilizations": {
+          "description": "Utilizations of managed instances inside the instance pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstancePoolManagedInstanceUtilization"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "InstancePoolManagedInstanceUtilization": {
+      "description": "A managed instance in an instance pool usage info.",
+      "type": "object",
+      "properties": {
+        "managedInstanceName": {
+          "description": "Name of the managed instance.",
+          "type": "string",
+          "readOnly": true
+        },
+        "currentValue": {
+          "format": "int32",
+          "description": "The current value.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "currentLimit": {
+          "format": "int32",
+          "description": "The current limit.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "requestedLimit": {
+          "format": "int32",
+          "description": "The requested limit.",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "InstancePoolUsage": {
```

Comment:
>InstancePoolUsage [](start = 5, length = 17)

and per spec, name is not just a string.. it is a json object.

"name": {
        "value": "Storage",
        "localizedValue": "Storage"
      }

Reply:
updated name to json object.